src/client/components/EditUserForm/UserRolePropsFields/UserRolePropsFields.tsx

```diff
@@ -16,7 +16,7 @@ type Props = {
 const UserRolePropsFields: React.FC<Props> = (props) => {
   const { role, onChange, enabled } = props
 
-  const { props: roleProps } = role
+  const { props: roleProps } = role ?? {}
```

Comment:
why there shouldn't be the role when this component is rendered ? doesn't it require the role?

Reply:
this will solve: if (!role) return null